examples/simple/src/main.rs

```diff
@@ -20,15 +20,14 @@ use winit::window::Window;
 use vello::wgpu;
 /// Simple struct to hold the state of the renderer
```

Comment:
I wouldn't normally describe an enum as a struct.

Reply:
I wouldn't either, this is just a residual comment lol.